[DOCS-7799]  Network Path - updated PR (#23772)
DataDog/documentation: content/en/network_monitoring/network_path/setup.md

---
title: Setup
kind: documentation
description: Setting up Network Path 
is_beta: true
further_reading:
- link: "/network_monitoring/network_path/list_view"
  tag: "Documentation"
  text: "Learn more about the List View in Network Path"
- link: "/network_monitoring/network_path/path_view"
  tag: "Documentation"
  text: "Learn more about the Path View in Network Path"
---

{{< site-region region="gov" >}}
<div class="alert alert-warning">Network Path for Datadog Network Performance Monitoring is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
{{< /site-region >}}

<div class="alert alert-info">Network Path for Datadog Network Performance Monitoring is in private beta. Reach out to your Datadog representative to sign up, and then use the following instructions to configure the Datadog Agent to gather network path data.</div>

> ## Setup Instructions on Linux Agent
> 
> 1. Agent version `7.55` or higher is required.

Reviewer: ```suggestion
## Overview

Setting up Network Path involves configuring your Linux environment to monitor and trace the network routes between your services and endpoints. This helps identify bottlenecks, latency issues, and potential points of failure in your network infrastructure.

## Setup

Agent version `7.55` or higher is required.

```

2. Enable [NPM][1].
3. Enable the `system-probe` traceroute module in `/etc/datadog-agent/system-probe.yaml` by adding the following:

```
traceroute:
  enabled: true

> ```

Reviewer: Indent this three spaces to the right.

> 4.  Enable network_path to monitor NPM connections by creating/editing the `/etc/datadog-agent/datadog.yaml` file: 

Reviewer: ```suggestion
4.  Enable `network_path` to monitor NPM connections by creating or editing the `/etc/datadog-agent/datadog.yaml` file: 
```

```

> Example:
> 

Reviewer: ```suggestion
```

network_path:
  connections_monitoring:
    enabled: true
  collector:
    workers: 10 # default 4

> ```

Reviewer: Indent this three spaces to the right.

> Detailed available configuration:

Reviewer: ```suggestion
For full configuration details, see the following:
```

```
network_path:
  connections_monitoring:
    ## @param enabled - bool - required - default:false
    ## Enable network path collection
    #
    enabled: true
  collector:
    ## @param workers - int - optional - default:4
    ## Number of workers that can collect paths in parallel
    ## Recommendation: leave at default
    #
    workers: 10

> ```

Reviewer: Indent this three spaces to the right.
